.gitignore

```diff
@@ -7,19 +10,23 @@
 /web/profiles/contrib/
 /web/libraries/
 
-# Ignore sensitive information
+# Ignore sensitive information.
 /web/sites/*/settings.php
-/web/sites/*/settings.local.php
```

Comment:
this and missed `services.local.yml` files are now managed with Drupal Scaffold `.gitignore` files created during `composer install`.